src/components/screens/wallet/votes/votes.js

```diff
@@ -37,11 +37,11 @@ const Votes = ({
 
   // Fetch delegate profiles to define rank, productivity and delegate weight
   useEffect(() => {
-    if (isEmpty(accounts.data) && votes.data.length) {
-      const addressList = votes.data.map(vote => vote.address);
+    const addressList = Object.keys(hostVotes);
+    if (isEmpty(accounts.data) && addressList.length) {
       accounts.loadData({ addressList, isDelegate: true });
     }
-  }, [votes.data]);
+  }, [hostVotes]);
```

Comment:
```suggestion
  }, [sentVotes]);
```

Reply:
@ManuGowda `hostVotes` returns both sent and received votes for the selected delegate, so it might be a more appropriate name.

Reply:
@gadjacobs This prop is defined as `hostVotes: state.voting` which only includes the votes sent by the account. not the ones they received.